BuildRelease.msbuild

```diff
@@ -120,6 +121,7 @@
 
         <NUnit ToolPath="$(NUnitToolsFolder)" Assemblies="Src\AutoFixture.NUnit2.UnitTest\bin\Release\Ploeh.AutoFixture.NUnit2.UnitTest.dll" ContinueOnError="true" />
         <NUnit ToolPath="$(NUnitToolsFolder)" Assemblies="Src\AutoFixture.NUnit2.Addins.UnitTest\bin\Release\Ploeh.AutoFixture.NUnit2.Addins.UnitTest.dll" OutputXmlFile="TestResult-Addins.xml" ContinueOnError="true" />
+		<Exec Command="$(NUnit3ToolsFolder)\nunit3-console.exe --framework=v4.0 --result=TestResult-NUnit3.xml Src\AutoFixture.NUnit3.UnitTest\bin\Release\Ploeh.AutoFixture.NUnit3.UnitTest.dll" ContinueOnError="true" />

```

Comment:
`ContinueOnError="true"` causes this step to only issue a warning instead of breaking the build. This means that even if a test fails, the build continues. This ought to instead be `ContinueOnError="false"`, so that a failing test breaks the build.